contracts/ProcedureTable.sol

```diff
@@ -132,7 +132,7 @@ library ProcedureTable {
         }
     }
 
-    function checkWriteCapability(Self storage self, uint192 key, uint256 toStoreAddress, uint256 reqCapIndex) internal view returns (bool) {
+    function checkWriteCapability(Self storage /* self */, uint192 key, uint256 toStoreAddress, uint256 reqCapIndex) internal view returns (bool) {
```

Comment:
Is this some sort of optimization?

Reply:
Reduces the warnings. Solidity doesn't allow you to turn off warnings for unused variables. Currently the only warning left is the empty struct (`Self`) in the procedure table, but I'm not sure how to tackle that one.